orchestration/dagster_orchestration/hca_orchestration/pipelines/cut_snapshot.py

```diff
@@ -136,15 +136,19 @@ def snapshot_start_notification(context: HookContext) -> None:
     required_resource_keys={'slack', 'snapshot_config', 'dagit_config'}
 )
 def snapshot_job_failed_notification(context: HookContext) -> None:
-    job_id = context.solid_output_values["result"]
+    if "result" in context.solid_output_values:
+        job_id = context.solid_output_values["result"]
+    else:
+        job_id = "NA"
```

Comment:
⛏️ _very_ small nitpick: `N/A` is more [conventional](https://en.wikipedia.org/wiki/N/A) for "not applicable"

Reply:
fixed! thank you